src/geocoder.jl

```diff
@@ -32,24 +95,29 @@ struct Geocoder
     end
 end
 
+function Geocoder(cities_data::AbstractDataFrame;
```

Comment:
yeap, adjusted this as well, so that the inner constructor is not specified and defaults, and other convenience constructors are now outer constructors

Reply:
Ah, so this is more a personal preference where I usually keep the default constructor to maintain maximum development flexibility.